blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/GenericElasticSearchIO.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2013 Rackspace
```

Comment:
Just noting that we are going to fix the copyright year for these files in another PR as a lot of files need that change and its easier to fix them all at once.
cc/ @tilogaat